server.js

```diff
@@ -54,27 +57,36 @@ app.post('/api/ticket', (req, res) => {
   let numberOfSeats = req.body.ticket.numberOfSeats;
   let sessionId = req.body.ticket.sessionId;
 
```

Comment:
What is the purpose of these variables if you're still using req.body.ticket.* below? Why using let? Maybe you can use single variable for ticket object from body? Or use object destructuring?